src/components/Banner/Banner.tsx

```diff
@@ -1,31 +1,30 @@
-import clsx from "clsx";
-import React, { ReactNode } from "react";
-import Button from "../Button";
-import Heading, { HeadingElement } from "../Heading";
-import Icon from "../Icon";
-import Text from "../Text";
-import colorStyles from "../common/Notifications/Notification.module.css";
-import styles from "./Banner.module.css";
+import clsx from 'clsx';
+import React, { ReactNode } from 'react';
+import styles from './Banner.module.css';
+import Button from '../Button';
+import Heading, { HeadingElement } from '../Heading';
+import Icon from '../Icon';
+import Text from '../Text';
 
-export type Variant = "brand" | "neutral" | "success" | "warning" | "error";
+export type Variant = 'brand' | 'neutral' | 'success' | 'warning' | 'error';
 
 export type BannerProps = {
   /**
    * A button or link that's placed in the banner separately from the main content.
    */
-  action?: React.ReactNode;
+  action?: ReactNode;
   /**
    * CSS class names that can be appended to the component.
    */
-  className?: string;
+  className?: ReactNode;
```

Comment:
`ReactNode` here seems weird. I think this is the only component doing this. I'll change this on a separate PR.

Reply:
good catch! thank you!